crates/aptos/src/move_tool/mod.rs

```diff
@@ -316,6 +316,23 @@ impl CliCommand<&'static str> for ProvePackage {
             install_dir: self.move_options.output_dir.clone(),
             ..Default::default()
         };
+
+        const APTOS_NATIVE_TEMPLATE: &[u8] = include_bytes!("aptos-natives.bpl");
+
+        let mut options = move_prover::cli::Options::default();
+        options.backend.custom_natives =
+            Some(move_prover_boogie_backend::options::CustomNativeOptions {
+                template_bytes: APTOS_NATIVE_TEMPLATE.to_vec(),
+                module_instance_names: vec![
+                    (
+                        "0x2::transfer".to_string(),
+                        "transfer_instances".to_string(),
+                    ),
+                    ("0x2::id".to_string(), "id_instances".to_string()),
+                    ("0x2::event".to_string(), "sui_event_instances".to_string()),
```

Comment:
Not entirely sure why these names are here...

Reply:
Oops. These came from Emma's example of using custom boogie files. I removed them.